packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -1153,9 +1153,9 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
 
         const offset = Vec2.from(event);
         const point = () => invertCoords(offset, context);
-        const textInputValue = this.textInput.getValue();
+        const { textInputValue, bbox } = this.textInput.getValue() ?? {};
```

Comment:
It is a little odd to return the bbox from a function called `getValue()`.

Perhaps would be nicer to make `getBBox()` public and just use `this.textInput.getBBox()`?